.expeditor/verify_private.pipeline.yml

```diff
@@ -114,12 +114,15 @@ steps:
             "/go/src/github.com/chef/automate"
           ]
 
-  - label: ":semgrep: Published"
+  - label: ":semgrep: Security"
```

Comment:
Changed label to be more meaningful. This is running a set of published checks, yes, but they are all about security.